test/catch-override-exception.test.js

```diff
@@ -20,7 +20,7 @@ test('catch exceptions in parent.override', (t) => {
     .start()
 
   app.ready(function (err) {
-    t.type(err, Error)
-    t.match(err, /catch it/)
+    t.assert.strictEqual(err.message, 'catch it')
+    testDone()
```

Comment:
Can we do this instead of `t.type`?

```suggestion
    t.assert.ok(err instanceof Error)
    t.assert.strictEqual(err.message, 'catch it')
    testDone()
```